### DISCUSSION TITLE
Need help understanding the purpose of related projects/services.

### DISCUSSION THREAD

**Discussion by donvi-bz** [iTwin/itwinjs-core#1580] [Q&A] [answered]
I've been trying to figure out if iTwin is a good fit for out purposes, and have started by just outlining what everything does. 

From my understanding, iTwin.js and iModel.js are essentially the same thing (though different than an iModel), and it can be used to display a static state of an iModel. iTwin should allow us to look around and see real time sensor data (?) as well information about physical properties. I can't really tell if it does much more than that, I've seen other features advertised on Bentley's website relating to Digital Twins, but I can't tell if that is separate.

Then it seems like there are two separate services offered by Bentley? One to host the iTwin (which it seems like we could do entirely locally if we want?) and another to store all the iModel information on the cloud? Also, how much can we accomplish purely locally?

If you could let me know if what I said is correct, that would be wonderful. Thanks in advance.

**Comment by mgooding** [comment, marked as answer]
> From my understanding, iTwin.js and iModel.js are essentially the same thing (though different than an iModel), and it can be used to display a static state of an iModel. iTwin should allow us to look around and see real time sensor data (?) as well information about physical properties. I can't really tell if it does much more than that, I've seen other features advertised on Bentley's website relating to Digital Twins, but I can't tell if that is separate.

You are largely correct. iModel.js was renamed to iTwin.js (though some npm package names remained the same). The "Capabilities" section on http://www.itwinjs.org is a good summary of the library's features.

[Our GitHub repositories](https://github.com/imodeljs/) and [YouTube channel](https://www.youtube.com/channel/UCp_6lNq7q6HW4r-Fa9I2v9g) also have good examples of projects implemented using iTwin.js.

> Then it seems like there are two separate services offered by Bentley? One to host the iTwin (which it seems like we could do entirely locally if we want?) and another to store all the iModel information on the cloud? Also, how much can we accomplish purely locally?

[iTwin Platform](https://developer.bentley.com/itwinplatform/) is our service offering that hosts your iModel and its information in the Bentley cloud, along with managing its timeline of change. The provided link describes the APIs that make up iTwin Platform.

You can also host iModels in your own cloud environment or use them locally on desktop or mobile. iTwin.js is a common library used for both strategies.

These options are one of the unique strengths of iTwin. You can use the iModel as your common persistence format while picking the appropriate tech stack for each part of your application (for example, mixing the Bentley-hosted web stack with Unity-based VR applications).
